schema/base.sql

```diff
@@ -6309,6 +6274,13 @@ ALTER TABLE ONLY rec_itunes
 ALTER TABLE ONLY rec_track
     ADD CONSTRAINT rec_track_digitisedby_fkey FOREIGN KEY (digitisedby) REFERENCES member(memberid) ON UPDATE CASCADE ON DELETE SET NULL;
 
+--
+-- Name: rec_track_lasteditedby_fkey; Type: FK CONSTRAINT; Schema: public; Owner: -
+--
+
+ALTER TABLE ONLY rec_track
+    ADD CONSTRAINT rec_track_lasteditedby_fkey FOREIGN KEY (last_edited_memberid) REFERENCES member(memberid) ON UPDATE CASCADE ON DELETE SET NULL;
```

Comment:
I agree with keeping the trail and not deleting members, but I do think more of the db should work without errors.